io/iouring-wrapper.cpp

```diff
@@ -244,6 +244,10 @@ class iouringEngine : public MasterEventEngine, public CascadingEventEngine {
             io_uring_prep_poll_multishot(sqe, fd_interest.fd, fd_interest.interest);
         }
         io_uring_sqe_set_data(sqe, &pair.first->second.io_ctx);
+        int ret = io_uring_submit(m_ring);
```

Comment:
We'd better to submit the SQEs just before waiting, in order to align with the batching behavior of the master engine. Low-latency mode can be added later, with in-kernel busy-polling.